server/tests/src/test/resources/database/db2.properties

```diff
@@ -2,9 +2,18 @@ id.column.type=VARCHAR(255)
 data.column.type=BLOB(1000)
 timestamp.column.type=BIGINT
 segment.column.type=BIGINT
-database.mode=EXTERNAL
+database.mode=CONTAINER
+database.container.name=ibmcom/db2
```

Comment:
Is this image available on quay.io? This will avoid issues with pull request limits on CI agents.